src/pages/users/users-page/users-page.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { push } from 'connected-react-router';
+import { Typography } from '@material-ui/core';
+import { useStyles } from './users-page.styles';
+import { getUsers, deleteUser } from '../../../redux/users/users.actions';
+import TableContainerRow from '../../../components/table-container-row';
+import TableContainerGenerator from '../../../components/table-container-generator';
+import { config } from '../../../configs';
+import LoadingBar from '../../../components/loading-bar';
+
+import { closeDialog } from '../../../redux/dialog-window/dialog-window.actions';
+import useSuccessSnackbar from '../../../utils/use-success-snackbar';
+import { refactorPhoneNumber } from '../../../services/refactor.service';
+
+const tableHeaders = config.tableHeadRowTitles.users;
+const { REMOVE_USER_TITLE } = config.buttonTitles;
+const { REMOVE_USER_MESSAGE } = config.messages;
+
+const UsersPage = (props) => {
+  const styles = useStyles();
+  const { openSuccessSnackbar } = useSuccessSnackbar();
+  const { list, loading } = useSelector(({ Users }) => ({
+    list: Users.list,
+    loading: Users.userLoading
+  }));
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getUsers());
+  }, [dispatch]);
+
+  const userDeleteHandler = (id) => {
+    const removeUser = () => {
+      dispatch(closeDialog());
+      dispatch(deleteUser(id));
+    };
+    openSuccessSnackbar(
+      removeUser,
+      REMOVE_USER_TITLE,
+      REMOVE_USER_MESSAGE,
+      REMOVE_USER_TITLE
+    );
+  };
+
+  const usersItems =
+    list !== undefined
```

Comment:
Does this check make any sense? Can `list` be null at all?

Reply:
fixed that